girder_worker/utils.py

```diff
@@ -153,6 +153,9 @@ def _flush(self):
 
             self._redirectPipes(True)
 
+    def flush(self):
+        self._flush()
```

Comment:
We may want to make this a no-op, depending on how often flush() is being called, so that it doesn't circumvent our rate limiting. The flushing at end-of-stream is already taken care of in `__exit__`.